teleport-node/update-necocli.sh

```diff
@@ -0,0 +1,20 @@
+#! /bin/sh -ex
+
+export HTTPS_PROXY="http://squid.internet-egress.svc:3128"
+
+DATE=$(curl -s "https://api.github.com/repos/cybozu-go/neco/releases/latest" | grep -Po '"tag_name": "\K.*?(?=")' | sed -e "s/release-//")
```

Comment:
In my opinion, using jq is more understandable.

```suggestion
DATE=$(curl -s "https://api.github.com/repos/cybozu-go/neco/releases/latest" | jq -r ".tag_name" | sed -e "s/release-//")
```